kntest/pkg/kubetest2/gke/options.go

```diff
@@ -34,10 +35,12 @@ func addOptions(gkeCmd *cobra.Command, cfg *kubetest2.GKEClusterConfig) {
 	f.IntVar(&cfg.MinNodes, "min-nodes", 1, "The minimum number of nodes.")
 	f.IntVar(&cfg.MaxNodes, "max-nodes", 3, "The maximum number of nodes.")
 	f.StringVar(&cfg.Network, "network", "e2e-network", "The network name for the GKE cluster.")
-	f.StringVar(&cfg.Version, "version", "latest", "The version of the GKE cluster.")
+	f.StringVar(&cfg.ReleaseChannel, "release-channel", "", "The release channel of the GKE cluster.")
+	f.StringVar(&cfg.Version, "cluster-version", "latest", "The version of the GKE cluster.")
```

Comment:
Notify this argument is mandatory if no release-channel is provided in the comment

Reply:
Actually this argument can be empty if no release-channel is provided, then it will use the default version, currently it's using `1.15.12-gke.2`, but I have added some comments to explain how it works with release-channel.